datalad/local/gitcredential_datalad.py

```diff
@@ -0,0 +1,174 @@
+import sys
+
+from datalad.downloaders import UserPassword
+from datalad.downloaders.providers import (
+    AUTHENTICATION_TYPES,
+    Providers,
+)
+from datalad.local.gitcredential import _credspec2dict
+from datalad.utils import Path
+
+git_credential_datalad_help = """\
+Git credential interface to DataLad's credential management system.
+
+In order to use this, one needs to configure git to use the credential helper
+'datalad'. In the simplest case this can be achieved by 'git config --add
+--global credential.helper datalad'. This can be restricted to apply to
+certain URLs only. See 'man gitcredentials' and
+http://docs.datalad.org/en/stable/credentials.html for details.
```

Comment:
http://docs.datalad.org/credentials.html would be better. It would take a while till that URL starts working.